nym-vpn-apple/ServicesMacOS/Sources/GRPCManager/GRPCManager.swift

```diff
@@ -40,6 +40,24 @@ public final class GRPCManager: ObservableObject {
         try? group.syncShutdownGracefully()
     }
 
+    // MARK: - Info -
+
+    public func version() async throws -> String {
+        try await withCheckedThrowingContinuation { continuation in
+            let call = client.info(Nym_Vpn_InfoRequest(), callOptions: CallOptions(logger: logger))
+
+            call.response.whenComplete { result in
+                switch result {
```

Comment:
You can return result directly with [`resume(with: Result<>)`](https://developer.apple.com/documentation/swift/checkedcontinuation/resume(with:)-3gh60)